causing/graph.py

```diff
@@ -6,10 +6,34 @@
 import numpy as np
 from numpy import allclose, isnan
 from causing import utils
-from causing.estimate import tvals
 from causing.model import Model
 
 
+def tvals(eff, std):
```

Comment:
I think, tvals should still be an estimate functionality instead of a graph functionality.

Reply:
It is only used in graphs.py and I don't want to put it into estimate.py, since if I import estimate, I have to also import pytorch. That would mean that the graph plotting requires pytorch (if I want to keep it simple).